setup.cfg

```diff
@@ -65,16 +68,16 @@ exclude =
 [options.extras_require]
 testing =
   # upstream
-  pytest >= 7
+  pytest >= 7.4
```

Comment:
LGTM! We are now dropping support for pre-3.9 (as part of #134) Makes sense to bump versions too.